A/ADIOS2/build_tarballs.jl

```diff
@@ -135,12 +131,14 @@ dependencies = [
     Dependency(PackageSpec(name="Blosc2_jll")),
     Dependency(PackageSpec(name="Bzip2_jll"); compat="1.0.8"),
     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"), v"0.5.2"),
-    # We cannot use HDF5 because we need an HDF5 configuration with MPI support
-    # Dependency(PackageSpec(name="HDF5_jll")),
+    Dependency(PackageSpec(name="HDF5_jll"); compat="~1.14"),
     Dependency(PackageSpec(name="SZ_jll")),
     Dependency(PackageSpec(name="ZeroMQ_jll")),
     Dependency(PackageSpec(name="libpng_jll")),
     Dependency(PackageSpec(name="zfp_jll")),
+
+    # LibCURL is required for HDF5 (at least) on x86_64-linux-musl-libgfortran5-cxx03-mpi+mpich
+    Dependency("LibCURL_jll"),
```

Comment:
I don't know if it's easier to delete system `libcurl`, to avoid it's picked up.

Reply:
Ooh, nice idea.